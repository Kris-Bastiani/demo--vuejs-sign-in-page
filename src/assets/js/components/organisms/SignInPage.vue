<template lang="pug">
	app-page
		.container.dsp__flex.pad__sml.w_max__reg
			.w__half
				h1.mrg_b__lrg Sign In

				login-form

			img.w__half(src='/assets/images/login.png', alt='')
</template>

<script>
import AppPage from '../molecules/AppPage.vue';
import LoginForm from '../molecules/LoginForm.vue';

export default {
	components: {
		AppPage,
		LoginForm,
	},
};
</script>
