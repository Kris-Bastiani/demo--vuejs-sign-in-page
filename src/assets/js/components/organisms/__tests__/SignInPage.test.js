import 'regenerator-runtime/runtime';
import { shallowMount } from '@vue/test-utils';
import SignInPage from '../SignInPage.vue';

describe('SignInPage', () => {
	it('Matches snapshot', () => {
		const wrapper = shallowMount(SignInPage);
		expect(wrapper.element).toMatchSnapshot();
	});
});
