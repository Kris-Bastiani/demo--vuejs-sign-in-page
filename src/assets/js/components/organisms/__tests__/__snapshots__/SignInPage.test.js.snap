// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInPage Matches snapshot 1`] = `
<app-page-stub>
  <div
    class="container flx_aln_y__center pad__sml w_max__reg"
  >
    <div
      class="w__full w__half__at_above_680"
    >
      <h1
        class="mrg_b__lrg"
      >
        Sign In
      </h1>
      <login-form-stub />
    </div>
    <img
      alt=""
      class="dsp__none__below_680 w__half"
      role="presentation"
      src="/assets/images/login.png"
    />
  </div>
</app-page-stub>
`;
